doc/architecture.md

```diff
@@ -33,41 +33,21 @@ Finally, the sinks are custom ones, one per each persistence element covered by
     * `NGSIKafkaSink`.
 * Twitter sinks:
     * `TwitterHDFSSink`.
- 
-![](./images/basic_architecture.jpg)
 
 [Top](#top)
 
 ## Advanced Cygnus architectures
-All the advanced archictures arise when trying to improve the performance of Cygnus. As seen above, basic Cygnus configuration is about a source writting Flume events into a single channel where a single sink consumes those events. This can be clearly moved to a multiple sink configuration running in parallel; there are several possibilities:
+It depends on the specific Cygnus agent you are going to use.
 
-### Multiple sinks, single channel
-You can simply add more sinks consuming events from the same single channel. This configuration theoretically increases the processing capabilities in the sink side, but usually shows an important drawback, specially if the events are consumed by the sinks very fast: the sinks have to compete for the single channel. Thus, some times you can find that adding more sinks in this way simply turns the system slower than a single sink configuration. This configuration is only recommended when the sinks require a lot of time to process a single event, ensuring few collisions when accessing the channel.
+In the case of Cygnus for NGSI, please check the [configuration examples](cygnus-ngsi/installation_and_administration_guide/configuration_examples.md). Because Flume/Cygnus configurations derive from Flume/Cygnus agent architecture, by checking the advanced configuration examples you'll be able to understand how complex Cygnus for NGSI architectures can be.
 
-![](./images/multiple_sinks_single_channel_architecture.jpg)
```

Comment:
The same images (and more) are available in text format in https://github.com/telefonicaid/fiware-cygnus/blob/master/doc/cygnus-ngsi/installation_and_administration_guide/configuration_examples.md